OpenRobertaServer/staticResources/js/app/roberta/controller/galleryList.controller.js

```diff
@@ -18,14 +18,47 @@ define([ 'require', 'exports', 'log', 'util', 'comm', 'message', 'guiState.contr
      */
     function init() {
 
+        initGalleryToolbar();
         initGalleryList();
         initGalleryListEvents();
         LOG.info('init gallery list view');
     }
     exports.init = init;
 
+    function getRobotGroups(){
```

Comment:
Please add here a TODO comment that we do not forget to add the real names of a group in the plugin properties to not have this workaround.